lib/postgrex/extensions/polygon.ex

```diff
@@ -0,0 +1,32 @@
+defmodule Postgrex.Extensions.Polygon do
+  @moduledoc false
+  import Postgrex.BinaryUtils
+  import Postgrex.Extensions.Point, only: [encode_point: 3]
+  use Postgrex.BinaryExtension, send: "poly_send"
+
+  def encode(type_info, polygon = %Postgrex.Polygon{vertices: vertices}, _, _)
+  when is_list(vertices) do
+    [
+      <<length(polygon.vertices)::int32>>,
+      Enum.map(polygon.vertices,
+        fn(point) -> encode_point(type_info, point, Postgrex.Polygon) end)
```

Comment:
Can you explain what you mean by "break this and assign to a variable" and why you want that?